core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -111,6 +113,7 @@ public ManipulationManager( Map<String, Manipulator> manipulators,
     /**
      * Determined from {@link Manipulator#getExecutionIndex()} comparisons during {@link #init(ManipulationSession)}.
      */
+    @Getter
```

Comment:
Sometimes it's better/safer to return `Collections.unmodifibleList()` if it's intended to be read-only rather than a direct reference and lombok could really make things easier if it supported such an option, but it doesn't, see <https://github.com/rzwitserloot/lombok/issues/1504>.

Reply:
Its an interesting point, but I'm not sure its really worth it.